libraries/dawg-common/src/main/java/com/comcast/video/dawg/common/MetaStb.java

```diff
@@ -559,13 +559,14 @@ public void setRackProxyUrl(String rackProxyUrl) {
      * {@inheritDoc}
      */
     public Boolean getRackProxyEnabled() {
-        return Boolean.parseBoolean((String) data.get(MetaStb.RACK_PROXY_ENABLED));
+        Object rpe = data.get(MetaStb.RACK_PROXY_ENABLED);
```

Comment:
setRackProxyEnabled stores the field as a boolean. So if you do a get after a set you will get a class cast exception.